pandas/io/formats/style.py

```diff
@@ -1211,13 +1229,13 @@ def css(x):
     def bar(
         self,
         subset=None,
-        axis=0,
+        axis: Optional[Axis] = 0,
         color="#d65f5f",
-        width=100,
-        align="left",
-        vmin=None,
-        vmax=None,
-    ):
+        width: float = 100,
```

Comment:
is it float or int?

Reply:
float, want me to assign it the value of  ```100.0```?